app/src/main/java/vn/edu/usth/minigh/fragments/IssuePrFragment.java

```diff
@@ -76,10 +77,18 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         else{
             image.setBackgroundResource(R.drawable.error);
         }
-        TextView name = view.findViewById(R.id.ghnameIP);
+        TextView name = view.findViewById(R.id.issuePrGhname);
         name.setText(mGHName);
-        TextView ipContent = view.findViewById(R.id.contentIP);
+        TextView ipContent = view.findViewById(R.id.issuePrContent);
         ipContent.setText(mContent);
+
+        LinearLayout ll = view.findViewById(R.id.issuesPrFrame);
+        ll.setTag(mGHName);
```

Comment:
What is this for?

Reply:
I forgot to change this. I'll remove it later ;;;